src/App.tsx

```diff
@@ -15,71 +16,95 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
```

Comment:
why do you reqrite component from functional to class?

Reply:
The main difference between a class component and functionality is the ability to save state. How can i change state of component without state?